changelogs/fragments/63140-acme-fix-fetch-url-status-codes.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- "ACME modules: properly handle fetch_url unsuccessful status codes"
```

That's a very technical description which won't help users at all. How about something like:
```suggestion
- "ACME modules: make sure some connection errors are handled properly"
```